### DISCUSSION TITLE
Infinite Truchet: Exercise 3

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#12] [Show and tell]
1. Open file `exercises/05-infinite-truchet/exercise-3.Rmd`
2. Run all of the code chunks in Part 1. This will give you access to the system `exercise_truchet_tiles()`.
3. Within `exercises/05-infinite-truchet/` there is an image `luft_13_roni.png`. Use https://mattdesl.github.io/colorgrab/  to select two colors to use in your system. 
4. Pass the hex codes for these colors to `color1` and `color2`. Modify the seed to experiment with different outputs. 
5.  Save your image using `ggsave()` code provided and share on this GitHub discussion

**Comment by roBertus31** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/25853405/931d09f4-2032-4e30-ae3c-34379dcdfee8)

**Comment by bshk93** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/65432465/d41c3137-1ed2-4986-a223-8615ad546f79)

**Comment by Kim-s-h** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/22897443/58824389-f4b5-4e53-b963-6ecc1767eed3)

**Comment by lwdozal** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/19821171/75cebaf4-8bc1-4f59-9c88-27d447636107)

**Comment by thomaspara** [comment]
![ex3-piece3](https://github.com/posit-conf-2023/creative-coding/assets/10082327/d96bc0df-6beb-47f5-a9a2-c259e5f61746)

**Comment by maxrogow** [comment]
![truchet](https://github.com/posit-conf-2023/creative-coding/assets/145362959/435e2027-43b8-4612-bd65-747a6372ee09)

**Comment by sshaughnessy** [comment]
![truchet_final](https://github.com/posit-conf-2023/creative-coding/assets/36434962/f42a94be-8f09-472b-ba27-57558528898c)

**Comment by shaunmwilkinson** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/145362966/cf6e3c48-8bfd-4958-9263-61a40606e432)

**Comment by natalieoshea** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/10425075/e63b4096-2c4d-4be0-af55-07e5ee55940c)

**Comment by dougtommet** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/7944225/68b1ec00-1e02-4215-abbb-b7b8513ad6c0)
